setup.py

```diff
@@ -97,16 +97,21 @@
             "pydivert>=2.0.3,<2.2",
         ],
         'dev': [
+            "awscli>=1.19,<2",
             "hypothesis>=5.8,<6.11",
             "parver>=0.1,<2.0",
             "pdoc>=4.0.0",
+            "pip-tools>=6.1.0,<6.2",
+            "pyinstaller==4.3",
             "pytest-asyncio>=0.10.0,<0.14,!=0.14",
             "pytest-cov>=2.7.1,<3",
             "pytest-timeout>=1.3.3,<2",
             "pytest-xdist>=2.1.0,<3",
             "pytest>=6.1.0,<7",
             "requests>=2.9.1,<3",
             "tox>=3.5,<4",
-        ]
+            "twine>=3.4.1,<4",
+            "wheel>=0.36.2,<0.37",
```

Comment:
These dependencies were specified in tox.ini, I've just moved them all in here for simplicity. Otherwise we need to maintain different pinned requirements files etc, it's all pain.